packages/block-library/src/style.scss

```diff
@@ -108,3 +108,14 @@
 .aligncenter {
 	clear: both;
 }
+
+// This resets the WordPress default
+img {
+	height: auto;
```

Comment:
Just a note for the `height:auto` here.
In the twentytwentyone theme this was causing a layout shift while the page was loading on slow connections. In order to fix that we had to add inline styles for the featured images (see https://github.com/WordPress/twentytwentyone/pull/692/files).

Reply:
@aristath Should that fix be baked into core then, and not just into this CSS reset?

If I recall correctly, this rule only became necessary after [lazy loading images launched](https://make.wordpress.org/core/2020/07/14/lazy-loading-images-in-5-5/).

Reply:
Yeah, it became an issue after lazy-loading. 
That function does the following:
* Gets the image width & height
* Calculates the aspect-ratio
* Adds `width:100%;height:X%` (X calculated from the aspect-ratio) to the image inline.

If `width:100%` is OK for all themes, then we can definitely port the fix to core, but I don't think all themes will want that...

Reply:
Ah ok, I see what you mean — a 100% width would probably cause its own set of problems in some themes. 😄

But a global fix of some kind seems to be a good idea.